### PR TITLE
Roles - added Meeting Scribe(s) section

### LINKED ISSUE #350: [Proposal] Add "scribe" role plus details to "Working group roles" document
Description: I'd like to add a "scribe" role plus details to the ["Working group roles" document](https://github.com/cncf/sig-security/blob/master/governance/roles.md).

Impact: Further fleshes out the roles documentation. This could be useful for members offering to volunteer as scribes during the weekly meetings.

Scope: Not much at all. The written portion would not exceed that for existing roles. I imagine proposed edits/additions in the pull request will consume more time than the initial commit.

TO DO
- [x] SIG Representative (@lumjjb)
- [x] Project leader(s)
- [ ] TBD

### PR DESCRIPTION
**Pull request by yeelinglam**
Fixed #350

### CODE REVIEW

File: governance/roles.md [hunk 1]
```diff
@@ -173,6 +173,21 @@ One or more of the following:
...
 
+### Meeting Scribe(s)
+
+Each meeting must have at a minimal one scribe and preferable with two.
+
+Prerequisites:
+* Prospect and/or active members (this is a good way to contribute initially)
+
+Scribe is expected to perform the following:
+* Review meeting Agenda with facilitator before start of meeting
```
[the remark this answers is not in the document]
Comment (yeelinglam) [reply governance/roles.md:181]: Shouldn't that be a part of the "wrap up" part for the facilitator to solicit for the next meeting?

[the remark this answers is not in the document]
Comment (lumjjb) [reply governance/roles.md:181]: Well scribe sign-up should be automatic each week like meeting facilitator, so this should be in the upcoming meetings section we have today where meeting facilitators are listed, and we can add a scribe entry as well

[the remark this answers is not in the document]
Comment (yeelinglam) [reply governance/roles.md:181]: Please do that and we could resolve this convo.

File: governance/roles.md [hunk 2]
```diff
@@ -173,6 +173,18 @@ One or more of the following:
...
 
+### Meeting Scribe(s)
+
+Each meeting must have at a minimal one scribe and preferable with two.
+
+Scribe is expected to perform the following:
+* Review meeting Agenda with facilitator before start of meeting
+* Summarize discussions and capture into meeting notes (not transcribe verbatim)
+* Clarify and annotate actions as discussion progresses with assignee and due dates
+* Partner with Facilitator to capture major occurring items in the meeting
```
Comment (lumjjb) [inline governance/roles.md:184]: Hmm, it seems a bit unclear how this should happen. Also, if we include meeting facilitator, we should add a bullet there to work with scribe as well

Comment (TheFoxAtWork) [reply governance/roles.md:184]: Concur - perhaps coordination of agenda with facilitator via slack #sig-security thread?  Maybe have the facilitator start the thread to request the scribe?

Comment (yeelinglam) [reply governance/roles.md:184]: I think the faciliator should start the thread to the scribe.  As for the scribe session, changing the first bullet to "Plan to coordinate/review meeting Agenda  with facilitator via Slack before start of meeting"?  I thought about putting "30mins beforehand", but might be best to leave it to Facilitator and Scribe to coordinate instead.

Comment (TheFoxAtWork) [reply governance/roles.md:184]: concur that sounds good.  @lumjjb ?

Comment (lumjjb) [reply governance/roles.md:184]: Yes that sounds good. Let's make that change and I think this should be good to merge

Comment (yeelinglam) [reply governance/roles.md:184]: Great, updated!

Comment (yeelinglam) [reply governance/roles.md:184]: Updated, please review and merge! ;)

**Comment by lumjjb** [comment]
Chair review @dshaw @pragashj @ultrasaurus

**Comment by yeelinglam** [comment]
@dshaw @pragashj @ultrasaurus - do you have any questions before approving?

**Comment by yeelinglam** [comment]
Hey @lumjjb - could you please merge this?  Thanks!

**Comment by lumjjb** [comment]
Merged! Thanks @yeelinglam !

**Comment by yeelinglam** [comment]
Thanks for the quick turnaround! @lumjjb

**Pull request merged**